presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/ObjectIdFunctions.java

```diff
@@ -65,8 +65,11 @@ public static Boolean equal(@SqlType("ObjectId") Slice left, @SqlType("ObjectId"
 
     @ScalarOperator(IS_DISTINCT_FROM)
     @SqlType(StandardTypes.BOOLEAN)
-    @SqlNullable
-    public static Boolean isDistinctFrom(@SqlType("ObjectId") Slice left, @IsNull boolean leftNull, @SqlType("ObjectId") Slice right, @IsNull boolean rightNull)
+    public static boolean isDistinctFrom(
```

Comment:
The formatting change makes it hard to see that the real change here is the removal of `@SqlNullable`

Reply:
Also, it's not clear how this "fixes IS DISTINCT for ...". Maybe add an explanation in the commit message describing how it was broken.

Reply:
> The formatting change makes it hard to see that the real change here is the removal of `@SqlNullable`

reverted



> Also, it's not clear how this "fixes IS DISTINCT for ...". Maybe add an explanation in the commit message describing how it was broken.

added explanation